### DISCUSSION TITLE
How to use extension components provided

### DISCUSSION THREAD

**Discussion by ZehuaZhang** [imzbf/md-editor-rt#67] [Q&A] [answered]
Is there a modular way or an api to use all or some of extensions provided in [component](https://github.com/imzbf/md-editor-rt/tree/docs/src/components)?

**Comment by imzbf** [comment, marked as answer]
Insert components through `defToolbars` and set index of components to `toolbars ` to specify where they should be displayed.

Perhaps what you need to refer to is the content [here](https://github.com/imzbf/md-editor-rt/blob/docs/src/pages/Preview/index.tsx#L94)
